mesh_cloud_server:
  sample_points: 100000